pkg/helm/client.go

```diff
@@ -56,8 +58,9 @@ const (
 )
 
 type CliClient struct {
-	version HelmVersion
-	helmBin string
+	version    HelmVersion
+	kubeClient kubernetes.Interface
```

Comment:
Is this still needed?

Reply:
good catch, I don't think we need this. I will update the PR.